src/Sulu/Bundle/TranslateBundle/Command/ExportCommand.php

```diff
@@ -76,6 +75,26 @@ protected function configure()
         );
     }
 
+    /**
+     * Detect public/web path.
+     *
+     * @param InputInterface $input
+     * @param OutputInterface $output
+     */
+    protected function interact(InputInterface $input, OutputInterface $output)
```

Comment:
Isn't the `interact` method meant to ask the user for stuff?

Reply:
hmm thats right - will move this logic to an own method and the run method